examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    hierarchical_ts = TSDataset(df=hierarchical_df, hierarchical_structure=hierarchical_structure, freq="MS")
</pre><p>Place the attributes in the order they appear in the dataset df, freq, hierarchical_structure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>